modules/initrd-boot-gui.nix

```
@@ -1,8 +1,22 @@
 { config, lib, pkgs, ... }:
 
 let
-  key-held = pkgs.runCommand "key-held.mrb" {} ''
-    ${pkgs.buildPackages.mruby}/bin/mrbc -o $out ${../boot/applets}/key-held.rb
+  minimalX11Config = pkgs.runCommandNoCC "minimalX11Config" {
+    allowedReferences = [ "out" ];
+  } ''
+    (PS4=" $ "; set -x
+    mkdir -p $out
+    cp -r ${pkgs.xlibs.xkeyboardconfig}/share/X11/xkb $out/xkb
+    cp -r ${pkgs.xlibs.libX11.out}/share/X11/locale $out/locale
+    )
```

```suggestion
    (
      PS4=" $ "; set -x
      mkdir -p $out
      cp -r ${pkgs.xlibs.xkeyboardconfig}/share/X11/xkb $out/xkb
      cp -r ${pkgs.xlibs.libX11.out}/share/X11/locale $out/locale
    )
```

JFYI I usually format subshells like this to make it more clear that they are one level lower.

---

Thanks, though I'm keeping it on the same level.

I would only indent shell when it's part of a control structure. Here it's really only to help with the verbosity of the build. Removing that subshell (including the initial in-line var and set) are intended to be a no-op. Since it wouldn't change the *actual* meaning of those commands, I personally think it's fine to do it that way.